packages/@apphosting/common/src/index.ts

```diff
@@ -1,5 +1,58 @@
 import { spawn } from "child_process";
 
+// Output bundle specifications to be written to bundle.yaml
+export interface OutputBundle {
+  version: "v1alpha";
+  serverConfig: ServerConfig;
+  metadata: Metadata;
+}
+
+// Fields needed to configure the App Hosting server
+interface ServerConfig {
+  // Command to start the server (e.g. ["node", "dist/index.js"]) assume this command is run from the root dir of the workspace
+  runCommand: string[];
+  // Environment variable present in the server execution environment.
+  environmentVariables?: EnvVarConfig[];
+  // The maximum number of concurrent requests that each server instance can receive.
+  // Defaults to 80.
+  concurrency?: number;
+  // The number of CPUs used in a single server instance.
```

Comment:
Are we intentionally diverging from our server docs?  https://cloud.google.com/run/docs/reference/rest/v2/projects.locations.services#resource:-service

Reply:
Oh good call, yeah this shouldn't diverge from those. Changed CPU to number